client/layout/universal-navbar-footer/index.tsx

```diff
@@ -1,10 +1,20 @@
 import './style.scss';
+import { getUrlParts } from '@automattic/calypso-url';
+import { getPathParts, getLanguage } from '@automattic/i18n-utils';
 import { useTranslate } from 'i18n-calypso';
 import SocialLogo from 'calypso/components/social-logo';
 import { navigate } from 'calypso/lib/navigate';
 
+function getRealPathName( path: string ) {
```

Comment:
This looks like a good candidate for extraction to a package - `@automattic/calypso-url` for example, or `@automattic/i18n-utils`.